examples/skills_tuning/instructlab/structured_summary.py

```
@@ -0,0 +1,216 @@
+# Load the flow
+flow = Flow(client).get_flow_from_file("flows/grounded_summary_extraction.yaml")
+
+# Initialize the synthetic data generator
+generator = SDG(flows=[flow])
+
+# %%
+seed_data = seed_data.select(range(10)) # note: this is just for demo purposes, in practice you can use the entire dataset
+generated_data = generator.generate(seed_data)
+
+# %%
+from rich.console import Console
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)